.github/workflows/ci.yml

```diff
@@ -1,42 +1,56 @@
 name: CI
 
 on:
-  # Run CI on all pushes.
   push:
-    branches:
-      - master
-
   pull_request:
-    branches:
-      - master
+
+  # Run CI daily even without code changes to ensure tests pass with updated dependencies.
+  schedule:
+    - cron: '0 0 * * 5'
+  # See:
+  # * https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#scheduled-events
+  # * https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07
+
+  # Allow triggering a CI run from the web UI.
+  workflow_dispatch:
+  # See:
+  # * https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  # * https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
 
 jobs:
   build-and-test:
-    name: Build and Test (Python ${{ matrix.python-version }} on ${{ matrix.os }})
-    runs-on: ubuntu-20.04
     strategy:
+      fail-fast: false
       matrix:
         os:
-          - ubuntu-20.04
-          - macos-latest
-          - windows-latest
-        python-version:
-          - 3.6
-          - 3.7
-          - 3.8
-          - 3.9
-
+        - { icon: 🐧, name: ubuntu-20.04 }
+        - { icon: 🍎, name: macos-latest }
+        #- { icon: 🧊, name: windows-latest }
```

Comment:
Add a comment above this line indicating that the tests are currently broken on Windows, and file an issue for it, please.

Reply:
Done. See #528.